iothub_client/tests/iothubtransport_ut/iothubtransport_ut.c

```diff
@@ -31,9 +31,6 @@ static void my_gballoc_free(void* ptr)
 
 #include "iothub_transport_ll.h"
 
-#define IOTHUB_TRANSPORT_H
```

Comment:
@jebrando long time no talk :)

Do you remember at all why this was done? At least the `_LL_H` macro would already be defined by the above include for `_ll.h`. The tests seem to pass.

Reply:
discussed offline